Library/Formula/kapacitor.rb

```diff
@@ -0,0 +1,191 @@
+    url "https://github.com/influxdb/enterprise-client.git",
+      :revision => "25665cba4f54fa822546c611c9414ac31aa10faa"
+  end
+
+  go_resource "github.com/influxdb/influxdb" do
+    url "https://github.com/influxdb/influxdb.git",
+      :revision => "45502d288f79b60051a7ec9260ad59dacb5b1a6c"
+  end
+
+  go_resource "github.com/twinj/uuid" do
+    url "https://github.com/twinj/uuid.git",
+      :revision => "89173bcdda19db0eb88aef1e1cb1cb2505561d31"
+  end
+
+  go_resource "golang.org/x/crypto" do
+    url "https://go.googlesource.com/crypto.git",
+      :revision => "beef0f4390813b96e8e68fd78570396d0f4751fc"
+  end
+
+  go_resource "gopkg.in/fatih/pool.v2" do
+    url "https://gopkg.in/fatih/pool.v2.git",
+      :revision => "cba550ebf9bce999a02e963296d4bc7a486cb715"
+  end
+
+  go_resource "github.com/BurntSushi/toml" do
+    url "https://github.com/BurntSushi/toml.git",
+      :revision => "056c9bc7be7190eaa7715723883caffa5f8fa3e4"
+  end
+
+  go_resource "github.com/kimor79/gollectd" do
+    url "https://github.com/kimor79/gollectd.git",
+      :revision => "61d0deeb4ffcc167b2a1baa8efd72365692811bc"
+  end
+
+  go_resource "gopkg.in/gomail.v2" do
+    url "https://gopkg.in/gomail.v2.git",
+      :revision => "41f3572897373c5538c50a2402db15db079fa4fd"
+  end
+
+  go_resource "github.com/serenize/snaker" do
+    url "https://github.com/serenize/snaker.git",
+      :revision => "d88cc11617e4f179fa2c821a60908d5fabd7bdad"
+  end
+
+  go_resource "github.com/shurcooL/markdownfmt" do
+    url "https://github.com/shurcooL/markdownfmt.git",
+      :revision => "c6ad3c69d14e5089336af1e41e595f0826df1b82"
+  end
+
+  go_resource "github.com/dustin/go-humanize" do
+    url "https://github.com/dustin/go-humanize.git",
+      :revision => "64dbdae0d393b7d71480a6dace78456396b55286"
+  end
+
+  go_resource "github.com/mattn/go-runewidth" do
+    url "https://github.com/mattn/go-runewidth.git",
+      :revision => "d96d1bd051f2bd9e7e43d602782b37b93b1b5666"
+  end
+
+  go_resource "github.com/russross/blackfriday" do
+    url "https://github.com/russross/blackfriday.git",
+      :revision => "300106c228d52c8941d4b3de6054a6062a86dda3"
+  end
+
+  go_resource "github.com/shurcooL/go" do
+    url "https://github.com/shurcooL/go.git",
+      :revision => "ee4d3ba2f39466dd17fc55dd260e89b8963640f8"
+  end
+
+  go_resource "github.com/shurcooL/sanitized_anchor_name" do
+    url "https://github.com/shurcooL/sanitized_anchor_name.git",
+      :revision => "10ef21a441db47d8b13ebcc5fd2310f636973c77"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    kapacitor_path = buildpath/"src/github.com/influxdb/kapacitor"
+    kapacitor_path.install Dir["*"]
+
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd kapacitor_path do
+      if build.head?
+        system "go", "install",
+               "-ldflags", "-X main.version 0.1.2-HEAD -X main.branch master -X main.commit #{`git rev-parse HEAD`.strip}",
```

Comment:
Maybe split out `git rev-parse HEAD` into a separate variable to make this line a bit more readable.